storage/blockdevice/COMPONENT_OSPIF/include/OSPIF/MX25LM51245G_config.h

```diff
@@ -26,20 +25,13 @@
 // The code below can be removed when users test with the new flash.
 #define NEED_DEFINE_SFDP_PARA
 
-#ifdef NEED_DEFINE_SFDP_PARA
-uint8_t _sfdp_head_table[32] = {0x53, 0x46, 0x44, 0x50, 0x06, 0x01, 0x02, 0xFF, 0x00, 0x06, 0x01,
-                                0x10, 0x30, 0x00, 0x00, 0xFF, 0xC2, 0x00, 0x01, 0x04, 0x10, 0x01,
-                                0x00, 0xFF, 0x84, 0x00, 0x01, 0x02, 0xC0, 0x00, 0x00, 0xFF
-                               };
-uint8_t _sfdp_basic_param_table[64] = {0x30, 0xFF, 0xFB, 0xFF, 0xFF, 0xFF, 0xFF, 0x1F, 0xFF, 0xFF,
-                                       0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x01, 0x14, 0xEC,
-                                       0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x0C, 0x20,
```

Comment:
I would think that values for this workaround is specific for each OSPI,
so this should be kept in this config file ?

Reply:
I agree with you. But it will report some errors when compiled if more than one file include this config file. _sfdp_head_table will be redefined.